<ui version="4.0" >
 <class>settingsDialog</class>
 <widget class="QDialog" name="settingsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="1" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabCeo" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>514</width>
        <height>384</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Geographic Settings</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Default Coordinate System:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="CSComboBox" >
         <item>
          <property name="text" >
           <string>NAD27 Lat/Lon</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>WGS84 Lat/Lon</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Default Declination:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="DeclinationLineEdit" >
         <property name="text" >
          <string>9.3</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Default UTM Zone</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QSpinBox" name="UTMZoneSpinBox" >
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="maximum" >
          <number>60</number>
         </property>
         <property name="value" >
          <number>13</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDF" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>514</width>
        <height>384</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>DF Fix Settings</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Minimum Fix Cut Angle:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QSpinBox" name="MinFCASpinBox" >
         <property name="maximum" >
          <number>180</number>
         </property>
         <property name="value" >
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAPRS" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>514</width>
        <height>384</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>APRS Settings</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>APRS Server IP Address:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="APRSServerLineEdit" >
         <property name="text" >
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>APRS Server Port:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="APRSServerPortLineEdit" >
         <property name="text" >
          <string>2023</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>APRS Call Sign:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="CallSignLineEdit" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>APRS CallPass:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="CallPassLineEdit" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>CSComboBox</tabstop>
  <tabstop>DeclinationLineEdit</tabstop>
  <tabstop>UTMZoneSpinBox</tabstop>
  <tabstop>MinFCASpinBox</tabstop>
  <tabstop>APRSServerLineEdit</tabstop>
  <tabstop>APRSServerPortLineEdit</tabstop>
  <tabstop>CallSignLineEdit</tabstop>
  <tabstop>CallPassLineEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
